db/migrate/20180130165200_drop_webmks.rb

```diff
@@ -0,0 +1,7 @@
+class DropWebmks < ActiveRecord::Migration[5.0]
+  class SettingsChange < ApplicationRecord; end
+
+  def up
+    SettingsChange.where(:resource_type => 'MiqServer', :key => '/server/remote_console_type', :value => 'MKS').delete_all
```

Comment:
We don't need the `:resource_type`.  Removing that from the scope would probably be a good idea in case someone added it to a zone settings.